system/mp/mpcomm.cpp

```diff
@@ -909,6 +953,10 @@ protected: friend class CMPPacketReader;
 #ifdef _TRACE
                 PROGLOG("MP: loopCnt start = %u", loopCnt);
 #endif
+                rd = 0;
+                byte replyBuf[sizeof(rd)];
```

Comment:
This (and elsewhere) should be sizeof(size32_t).  It should match the type of whatever item the data is then being memcpy'd into.

Reply:
ok, changed